### DISCUSSION TITLE
没法下载支持库

### DISCUSSION THREAD

**Discussion by ghost** [Meloong-Git/PCL#1818] [提问] [answered]
为什么一启动游戏就会提示这个，并且因为这个我也无法下载游戏版本，也用过管理员权限和系统权限均无法下载或补齐
<img width="645" alt="屏幕截图 2023-05-10 215543" src="https://github.com/Hex-Dragon/PCL2/assets/133128438/80efced6-4e29-4a2e-aebd-c1d2b0adca74">

**Comment by WForst-Breeze** [comment]
和 #957 类似，不过该 Issue 并未提供后续解决办法，请尝试以下方法，如仍无法解决问题请补充相应信息：
请检查您相应文件夹的权限是否进行了有效配置，即您的用户是否拥有目标文件夹的读写权限。

**Comment by LTCatt** [comment, marked as answer]
这是由于访问被系统拒绝，PCL 无法处理，也无法修复。
尝试删除那个文件或文件夹再试。
